src/main/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtil.java

```diff
@@ -408,4 +408,44 @@ else if (sibling.getType() == TokenTypes.SINGLE_LINE_COMMENT) {
                 || BlockCommentPosition.isOnPackage(blockComment));
     }
 
+    /**
+     * Gets the content of inline custom tag.
+     *
+     * @param javadocInlineTag javadoc inline tag node.
+     * @return String consisting of the content of inline custom tag.
+     */
+    public static String getContentOfInlineCustomTag(DetailNode javadocInlineTag) {
+        final DetailNode[] childrenOfJavadocInlineTag = javadocInlineTag.getChildren();
+        DetailNode nodeToBeProcessed = childrenOfJavadocInlineTag[2];
+        final DetailNode endingNode =
+                childrenOfJavadocInlineTag[childrenOfJavadocInlineTag.length - 1];
+        final StringBuilder customTagContent = new StringBuilder(256);
+
+        while (nodeToBeProcessed != endingNode) {
+            extractContentInlineCustomTag(nodeToBeProcessed, customTagContent);
+            nodeToBeProcessed = getNextSibling(nodeToBeProcessed);
+        }
+        return customTagContent.toString();
+    }
+
+    /**
+     * Extracts the content of inline custom tag recursively.
+     *
+     * @param node DetailNode
+     * @param customTagContent content of custom tag
+     */
+    private static void extractContentInlineCustomTag(DetailNode node,
```

Comment:
Since this is a private method used as helper for `getContentOfInlineCustomTag`, can we just call it `extractContent`? Or maybe `extractInlineTagContent`?

Reply:
Renamed to `extractInlineTagContent`

Reply:
Resolved.